src/mantarray_desktop_app/sub_processes/file_writer.py

```diff
@@ -1257,6 +1257,7 @@ def _process_update_name_command(self, comm: Dict[str, str]) -> None:
         """Rename recording directory and h5 files to kick off auto upload."""
         # only perform if new name is different from the original default name
         if self._current_recording_dir == comm["default_name"] != comm["new_name"]:
+            comm["new_name"] = comm["new_name"].strip()
```

Comment:
We might want to strip this in the route that receives this new name. Also we always want to unit test changes we make. I know we aren't doing that in the cloud right now, but pushing updates for the desktop app is much more work than in the cloud so it's much more important to make sure everything is adequately tested before we ship